### PR TITLE
[DO-1234] New Manager for DataOcean model

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: data_ocean/models.py [hunk 1]
```diff
@@ -3,11 +3,20 @@
 from django.utils.translation import gettext_lazy as _
 
 
+class DataOceanManager(models.Manager):
+    # exclude soft-deleted objects from queryset
+    def get_queryset(self):
+        return super().get_queryset().exclude(deleted_at__isnull=False)
```
Comment (roman-tiukh) [inline data_ocean/models.py:9]: NOT (deleted_at NOT IS NULL) ?
```suggestion
        return super().get_queryset().filter(deleted_at__isnull=True)
```
deleted_at IS NULL

File: data_ocean/models.py [hunk 2]
```diff
@@ -3,11 +3,20 @@
...
+
+
 class DataOceanModel(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True, null=True, blank=True)
     deleted_at = models.DateTimeField(null=True, blank=True, default=None)
 
+    objects = DataOceanManager()
+    include_deleted_objects = models.Manager()
```
Comment (roman-tiukh) [inline data_ocean/models.py:18]: long manager name.
try something like `all_objects`

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: data_ocean/models.py (hunk 1 above)
Comment (IegorSoboliev) [reply data_ocean/models.py:9]: first query: cost=0.29..21854.25 rows=45605 width=1185
second guery: cost=0.29..21854.25 rows=45605 width=1185

File: data_ocean/models.py (hunk 2 above)
Comment (IegorSoboliev) [reply data_ocean/models.py:18]: I think this is more clear name

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**